niworkflows/viz/utils.py

```diff
@@ -199,20 +199,31 @@ def extract_svg(display_object, dpi=300, compress='auto'):
 def cuts_from_bbox(mask_nii, cuts=3):
     """Finds equi-spaced cuts for presenting images"""
     from nibabel.affines import apply_affine
-    mask_data = mask_nii.get_data()
-    B = np.argwhere(mask_data > 0)
-    start_coords = B.min(0)
-    stop_coords = B.max(0) + 1
+
+    mask_data = mask_nii.get_data() > 0.0
+    ijk_counts = [
+        mask_data.sum(2).sum(0),  # project to i
+        mask_data.sum(2).sum(1),  # project to j
+        mask_data.sum(1).sum(0),  # project to k
+    ]
+
+    ijk_th = [
+        int((mask_data.shape[1] * mask_data.shape[2]) * 0.2),   # sagittal
+        int((mask_data.shape[0] * mask_data.shape[2]) * 0.0),   # coronal
+        int((mask_data.shape[0] * mask_data.shape[1]) * 0.3),   # axial
+    ]
```

Comment:
This is opaque to me. `ijk_th` reads like threshold, but I don't see how this should produce a threshold.

Reply:
Added some comments :+1: